portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,21 @@
   </head>
   <body>
     <div id="content">
-    <h1>My Portfolio</h1>
-    <p>Hi! I'm Lily, a STEP Intern at Google this summer and a rising junior @ Stanford studying CS + English.</p>
-    <img src="/images/flower.jpg" alt="Red Flower">
-    <p>Click here to get a random message:</p>
-    <button onclick="addRandomGreeting()">Hello</button>
-    <div id="greeting-container"></div>
+      <h1>Lily Zhou</h1>
+      <div class="icon-row">
+        <a href="https://www.linkedin.com/in/lily-z-408069122"><img src="/images/linkedin.jpg" width="30" height="30" alt="LinkedIn"></a>
+        <a href="https://github.com/lilymzhou"><img src="/images/github.jpg" width="30" height="30" alt="GitHub"></a>
+      </div>
+      <p>Hi! I'm Lily, a STEP Intern at Google this summer and a rising junior @ Stanford studying CS + English.</p>
+      <img src="/images/flower.jpg" alt="Red Flower">
+      <p>Currently:</p>
+      <ul>
+        <li>reading <i>Interior Chinatown</i> by Charles Yu</li>
+        <li>based in Foster City, California</li>
+        <li>learning how to bake</li>
+      </ul>
+      <p>Click <a href="images.html">here</a> to view my food journal.</p>
```

Comment:
If the user is using a screen reader and cycling through the set of links on the page, this one will just be read as "here", which lacks context. You can make this more accessible by setting the `aria-label` attribute to be something like "Link to food journal", or by rephrasing the item itself, e.g. `<a>My food journal</a>`.

More info: https://www.washington.edu/accessibility/links/